test/ctia/auth/jwt_test.clj

```diff
@@ -41,12 +41,18 @@
            :delete-casebook}
          (sut/scope-to-capabilities (sut/casebook-root-scope)))
       "Check the casebook capabilities from the casebook scope")
-  (is  (= #{:developer :specify-id :external-id}
-          (set/difference caps/all-capabilities
-                          (sut/scopes-to-capabilities #{(sut/entity-root-scope)
-                                                        (sut/casebook-root-scope)})))
-       "with all scopes you should have most capabilities except some very
+  (is (= #{:developer :specify-id :external-id}
+         (set/difference caps/all-capabilities
+                         (sut/scopes-to-capabilities #{(sut/entity-root-scope)
+                                                       (sut/casebook-root-scope)})))
+      "with all scopes you should have most capabilities except some very
        specific ones")
+
+  (is (= #{:developer :specify-id :external-id}
+         (set/difference caps/all-capabilities-no-casebook
+                         (sut/scopes-to-capabilities #{(sut/entity-root-scope)})))
+      "entity scope don't provide casebook")
```

Comment:
Would you mind to add another test to see the difference between `caps/all-capabilities` when the scope doesn't contain casebook so we could verify the difference between `all-capabilities` and `all-capabilities-no-casebook`.